utilities/load-test/loadtest/loadtest_test.go

```diff
@@ -2,38 +2,355 @@ package loadtest
 
 import (
 	"context"
-	"encoding/json"
 	"fmt"
+	"os/user"
+	"reflect"
 	"testing"
+	"time"
 
-	// appv1 "github.com/argoproj/argo-cd/v2/pkg/apis/application/v1alpha1"
+	appv1 "github.com/argoproj/argo-cd/v2/pkg/apis/application/v1alpha1"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func TestAThing(t *testing.T) {
+// type appResource struct{
+// 	AppInfo *appv1.Application
+// 	Count int
+// }
 
-	// GetE2EEFixtureK8sClient is from the ApplicationSet repo.
-	// You can see more examples of how to use it, here: https://github.com/argoproj-labs/applicationset/search?q=GetE2EFixtureK8sClient
+var (
+	// Check for an API (For now it is asume that the kubeconfig og each user lies in ~/.kube/config)
+	userLocal, _ = user.Current()
+	kubeconfig   = userLocal.HomeDir + "/.kube/config"
+	master       = ""
+	namespace    = "argocd"
+	podName      = ""
+)
+
+func TestNGuestbook(t *testing.T) {
+	// TEST 1: Deply 100 guestbook in different namespace
+	fmt.Print("Testing for 100 Guestbook applications\n")
 
-	// In this example, I am retrieving all Application CRs within the 'argocd' namespace
+	fmt.Print("\n\nPods Memory Resource before creation:\n\n")
+	memoryInit := Get_pod_info(kubeconfig, master, namespace, podName)
+	PodInfoParse(memoryInit)
 
-	appList, err := GetE2EFixtureK8sClient().AppClientset.ArgoprojV1alpha1().Applications("argocd").List(context.TODO(), v1.ListOptions{})
+	// Get Pod Restart Count
+	PodC := PodRestartcount(namespace)
+
+	for i := 1; i < 101; i++ {
+		guestbookApp := &appv1.Application{
+			TypeMeta: v1.TypeMeta{
+				Kind:       "Application",
+				APIVersion: "argoproj.io/v1alpha1",
+			},
+			ObjectMeta: v1.ObjectMeta{
+				Name:      "guestbook-" + fmt.Sprintf("%d", i),
+				Namespace: "argocd",
+			},
+			Spec: appv1.ApplicationSpec{
+				Source: appv1.ApplicationSource{
+					Path:           "guestbook",
+					TargetRevision: "HEAD",
+					RepoURL:        "https://github.com/argoproj/argocd-example-apps.git",
+				},
+				Destination: appv1.ApplicationDestination{
+					Namespace: "guestbook-" + fmt.Sprintf("%d", i),
+					Server:    "https://kubernetes.default.svc",
+				},
+				Project: "default",
+			},
+		}
+
+		_, err_create := GetE2EFixtureK8sClient().AppClientset.ArgoprojV1alpha1().Applications(namespace).Create(context.TODO(), guestbookApp, v1.CreateOptions{})
+		if err_create != nil {
+			t.Errorf("error, %v", err_create)
+			return
+		}
+	}
 
-	if err != nil {
-		t.Errorf("error, %v", err)
+	// Check if the Pod is getting refreshed or not
+	if reflect.DeepEqual(PodC, PodRestartcount(namespace)) != true {
+		t.Errorf("error, %v", "The Pods are getting refreshed!")
 		return
 	}
 
-	for _, app := range appList.Items {
+	fmt.Print("\n\nPods Memory Resource after Creation:\n\n")
```

Comment:
We should move the pod memory resource check after the `wait.Poll`, which I talk about below.